docs/patterns/single-new-eks-observability-accelerators/single-new-eks-gpu-opensource-observability.md

```diff
@@ -0,0 +1,260 @@
+# Single New EKS Cluster Open Source Observability Accelerator for NVIDIA GPU-based clusters
+
+Graphics Processing Units (GPUs) play an integral part in the Machine Learning (ML) workflow, by providing the scalable performance needed for fast ML training and cost-effective ML inference. On top of that, they are used in flexible remote virtual workstations and powerful HPC computations.
+
+This pattern shows you how to monitor the performance of the GPUs units, used in an Amazon EKS cluster leveraging GPU-based instances.
+
+Amazon Managed Service for Prometheus (AMP) and Amazon Managed Grafana (AMG) are open source tools used in this pattern to collect and visualise metrics respectively.
```

Comment:
AMP and AMG are not official names of those managed OSS products, i used it AMP and AMG just for short form for PR. Lets remove AMP and AMG references and call it with full names.

Reply:
they actually are :) ?
see here:
https://aws.amazon.com/about-aws/whats-new/2021/05/aws-announces-a-price-reduction-for-amazon-managed-service-for-prometheus-amp/
https://aws.amazon.com/grafana/sla/

Reply:
These docs needs to be updated, Service team does not want to reference these services as AMP or AMG. I have got shot multiple times in blogs. So lets change it

Reply:
done